index.html

```diff
@@ -559,6 +559,22 @@ <h3>Use Cases and Requirements</h3>
 or <a>verifiers</a>, because the conformance of ecosystem roles are highly
 application, use case, and market vertical specific.
         </p>
+
+        <p>
+While digital proofs, a subset of which are digital signatures, are required
+to ensure the protection of a <a>verifiable credential</a>, this specification
+does not standarize on any single digital signature format. At the time of
+publication, Working Group members had implemented
+<a>verifiable credentials</a> using at least three proof mechanisms:
+JSON Web Tokens [[JWT]], Linked Data Signatures [[?LD-SIGNATURES]], and
+Camenisch-Lysyanskaya Zero-Knowledge Proofs [[?CL-SIGNATURES]]. The group
+expects some of these mechanisms, as well as new ones, to mature independently
+and become standardized in time. One of the goals of this specification is to
+provide a data model that can be protected by a variety of current and future
+digital proof mechanisms. Conformance to this specification does not
+depend on the details of a particular proof mechanism; it requires clearly
```

Comment:
This makes it sound like the proof doesn't matter, where in practice the proofing mechanism for the credential is what makes it trustable within a real system. This important aspect of VC's might get lost by someone reading this paragraph. I would recommend adding text that having and validating a proof are an essential part of processing a VC but that the details of validating or applying the proof/signature are outside the scope of this specification.

Reply:
Fixed in 56a7b6b382369b68f345ec8c854ed64c12b91f3f.